sharding-jdbc/sharding-jdbc-core/src/main/java/org/apache/shardingsphere/shardingjdbc/jdbc/refreh/AbstractTableStatementMetaData.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.shardingsphere.shardingjdbc.jdbc.refreh;
+
+import org.apache.shardingsphere.core.metadata.ShardingMetaDataLoader;
+import org.apache.shardingsphere.core.metadata.ShardingTableMetaDataDecorator;
+import org.apache.shardingsphere.core.rule.ShardingRule;
+import org.apache.shardingsphere.encrypt.metadata.EncryptTableMetaDataDecorator;
+import org.apache.shardingsphere.shardingjdbc.jdbc.core.context.ShardingRuntimeContext;
+import org.apache.shardingsphere.sql.parser.binder.metadata.table.TableMetaData;
+import org.apache.shardingsphere.underlying.common.config.properties.ConfigurationPropertyKey;
+
+import java.sql.SQLException;
+
+/**
+ * The type Abstract table statement meta data.
+ */
+public abstract class AbstractTableStatementMetaData {
+    
+    /**
+     * Load table meta table meta data.
+     *
+     * @param tableName              the table name
```

Comment:
Please reference other javadoc and make them consist

Reply:
fix it

Reply:
It looks like not fixed yet.